bin/cli.js

```diff
@@ -195,6 +195,21 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('migrate:printAll')
+    .description('Print all migrations on directory')
+    .option('--direction', 'Specify migration direction to print')
+    .action(function() {
+      const direction = argv.direction || 'all';
```

Comment:
```suggestion
      const { direction } = argv;
```
you can use `undefined` as "all" and it makes easier to the user, because --direction can be an optional value

Reply:
At this moment, direction is already optional. The value 'all' is used when doesn't have direction.
Maybe is good to have an direction validation to check if is up or Down what you think?

Reply:
ok! Yeah, it's good to have this validation